libraries/browser-tracker-core/src/tracker/index.ts

```diff
@@ -610,7 +604,7 @@ export function Tracker(
       deleteCookie(sesname, configCookieDomain, configCookieSameSite, configCookieSecure);
       if (!configuration?.preserveSession) {
         memorizedSessionId = uuid();
-        memorizedVisitCount = 0;
+        memorizedVisitCount = 1;
```

Comment:
I'm so confused why was this set to 0 before, it seems that it was expected that right after `clearUserData()` is called there would be a new session created that would increment it to 1, but I don't see how that could happen 🤷  It looks good, just makes me a bit nervous that I don't understand the original intention.